test/summarize_test/summarize_test.py

```diff
@@ -0,0 +1,43 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: MIT-0
+
+import unittest
+import unittest.mock
+from unittest.mock import call, patch
+
+import json
+import pathlib
+import sys
+
+sys.path.append("../../src/cbmc_starter_kit/template-for-repository/proofs/lib")
+import summarize
+
+
+class TestSummarizeResults(unittest.TestCase):
+    def setUp(self):
+        self.run_file = pathlib.Path().cwd() / "sample_run.json"
+
+
+    @patch('builtins.print')
+    def test_print_proof_results(self, mock):
+        summarize.print_proof_results(self.run_file)
+        expected_calls = []
+        expected_calls.append(
+            call(
```

Comment:
I'm not going to block on this, but I find un-namespaced function calls to be slightly distracting. I don't see a good reason that this shouldn't be `unittest.mock.call`

Reply:
I used `unittest.mock.call` to be more clear.